client/package.json

```diff
@@ -16,6 +16,7 @@
     "axios": "^0.24.0",
     "bootstrap": "^5.1.3",
     "hamburger-react": "^2.4.1",
+    "lodash": "^4.17.21",
```

Comment:
Where lodash is needed?

Reply:
Forgot to remove it after testing something, I will remove it now.